python/mxnet/metric.py

```diff
@@ -475,8 +475,84 @@ def update(self, labels, preds):
             self.num_inst += num_samples
 
 
+class _BinaryClassificationMixin(object):
+    """
+    Private mixin for keeping track of TPR, FPR, TNR, FNR counts for a classification metric.
```

Comment:
Could you add more explanation of what this does?

Reply:
Updated. Let me know if that's not what you had in mind.